packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Arc/index.js

```diff
@@ -20,6 +20,7 @@ import { t } from '@superset-ui/translation';
 import { ChartMetadata, ChartPlugin } from '@superset-ui/chart';
 import thumbnail from './images/thumbnail.png';
 import transformProps from '../../transformProps';
+import controlPanel from '../../controlPanels/DeckArc';
```

Comment:
NIT/curious as to your thoughts: would it be easier to add/remove/maintain these "layers" if the controls were alongside each layer/plugin, i.e. renaming them all controls.js, and then including them here with `import controlPanel from './controls';`?

Reply:
yes that would easier and less code!